03_Data_Management/General.Rmd

```diff
@@ -8,15 +8,47 @@ knitr::include_graphics("images/network/Data Management_General.png")
 
 ### Sort
 
+Sorting out features in ascending or descending order seems a quite primitive operation in any programming language or software package. Even though it is indeed primitive, it is also quite important for filtering and cleaning our datasets. In ArcGIS pro this operation is performed using the tool `Sort`, which is part of the General toolset of the Data Managenent Toolbox. Below we present how we can perform the aforementioned operation using R. For our example we use the Simple Feature object _bezirke_, which depicts the districts within the country of Switzerland Furthermore for performing the sorting operation, we use the column that represents the area (in square km) of every of the districts. 
```

Comment:
`bezirke`

Reply:
@ratnanil  the correct is in bold -- since it is a dataset name -- I added it.